src/script_noxscript_builtins.go

```diff
@@ -1754,6 +1747,34 @@ func nox_script_getNextInvItem_5138E0() int {
 	return 0
 }
 
+func zombieSetStayDead(obj *Object) {
+	if obj != nil {
+		if obj.Class().Has(object.ClassMonster) {
+			*(*uint32)(unsafe.Add(obj.updateDataPtr(), 1440)) |= 0x100000
```

Comment:
You could use specific data type here (there's a specific one for each `obj.Class()`):
```suggestion
			obj.AsUnit().updateDataMonster().field_360 |= 0x100000
```

Reply:
Thanks for the suggestion. Simply didn't know there's a monster model